UIKITDZ_Clone_Instagram/UIKITDZ_Clone_Instagram/Cell/ProfileCells/ProfileHistory/CollectionViewCell/HistoryArhivCollectionViewCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  HistoryArhivCollectionViewCell.swift
+//  1234
+//
+//  Created by Alexander Kokh on 23.07.2021.
+//
+
+import UIKit
+
+final class HistoryArhivCollectionViewCell: UICollectionViewCell {
+
+    // MARK: - IBOutlet
+    @IBOutlet weak var historyLabel: UILabel!
+    @IBOutlet weak var historyImage: UIImageView!
```

Comment:
```suggestion
    @IBOutlet weak var historyImageView: UIImageView!
```
почему Image ? это же ImageVeiw